backend/conftest.py

```diff
@@ -49,3 +49,16 @@ def fixture_mock_centrifugo(mocker):
         side_effect=async_mock_centrifugo,
     )
     return async_mock_centrifugo
+
+
+@pytest.fixture(name="mock_dataStorage_read")
```

Comment:
This whole file have been moved inside the tests folder and you no longer need to write different fixtures for different endpoints.

Reply:
Yeah I noticed that boss. Moved already